src/components/NavBar/NavBar.js

```diff
@@ -1,34 +1,60 @@
 import React from 'react'
 import styled from 'styled-components/macro'
+import { getTimecodeTotalLengthFromSequenceCards } from '../../utils/Timecode'
+import { Link, useLocation } from 'react-router-dom'
 
 export default function NavBar({
   firstHalfTimeCode,
   secondHalfTimeCode,
   interviewTimeCode,
   countSpecials,
 }) {
+  const location = useLocation()
   return (
     <NavWrapper>
       <ul>
```

Comment:
In my app, I cut out the `ul` and `li`, might save you some unnecessary lines of code